presto-docs/src/main/sphinx/release/release-0.133.rst

```diff
@@ -16,3 +16,5 @@ General Changes
   ``dictionary_aggregation`` session property.
 * Fix race which could cause queries to fail when using :func:`concat` on
   :ref:`array_type`, or when enabling ``columnar_processing_dictionary``.
+* Add sticky headers and the ability to sort the tasks table on the query page
```

Comment:
Maybe flip the sentence order:

"Update query page in the web interface to have sticky headers and the ability to sort the tasks table."